main.tf

```diff
@@ -11,6 +11,40 @@ resource "ibm_resource_instance" "scc_instance" {
   tags              = var.resource_tags
 }
 
+resource "ibm_scc_provider_type_instance" "scc_provider_type_instance_instance" {
+  depends_on       = [ibm_iam_authorization_policy.scc_wp_s2s_access]
+  count            = var.attach_wp ? 1 : 0
+  instance_id      = ibm_resource_instance.scc_instance.guid
```

Comment:
It says ID, so maybe change to `ibm_resource_instance.scc_instance.id` (its probably just the guid anyway)

Reply:
I did try with `id` originally, but it didn't work. I guess the same for cos auth policy https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/c7a2a117af496e4167d966c1d1480315e1235d42/main.tf#L54